src/inet/tests/TestInetCommonPosix.cpp

```diff
@@ -454,8 +452,11 @@ void ServiceEvents(struct ::timeval & aSleepTime)
         }
     }
 
+    gSystemLayer.StartTimer(
```

Comment:
a comment of why we add a noop timer would be nice (like 'we use it as a wakeup trigger')

Reply:
Will do. I'm going to temporarily convert this PR to draft because I've just noticed that the callback there conflicts with #8907.